packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -403,7 +403,7 @@ class ServiceFirmwareUpdate extends ServiceBase {
       hasUpgrade,
       isBootloaderMode: features
         ? (await deviceUtils.getDeviceModeFromFeatures({ features })) ===
-          EOneKeyDeviceMode.bootloader
+        EOneKeyDeviceMode.bootloader
```

ESLint

[error] Insert `··`

(prettier/prettier)